src/electron/views/virtual-keyboard/virtual-keyboard-buttons.tsx

```diff
@@ -3,15 +3,27 @@
 
 import { NamedFC } from 'common/react/named-fc';
 import { NarrowModeStatus } from 'DetailsView/components/narrow-mode-detector';
+import { DeviceFocusControllerFactory } from 'electron/platform/android/device-focus-controller-factory';
+import { AdbWrapperHolder } from 'electron/platform/android/setup/adb-wrapper-holder';
 import * as styles from 'electron/views/virtual-keyboard/virtual-keyboard-buttons.scss';
 import { Button, css, Icon } from 'office-ui-fabric-react';
 import * as React from 'react';
 
-export type VirtualKeyboardButtonsProps = { narrowModeStatus: NarrowModeStatus };
+export type VirtualKeyboardButtonsDeps = {
+    deviceFocusControllerFactory: DeviceFocusControllerFactory;
+    adbWrapperHolder: AdbWrapperHolder;
+};
+
+export type VirtualKeyboardButtonsProps = {
+    deps: VirtualKeyboardButtonsDeps;
+    narrowModeStatus: NarrowModeStatus;
+    deviceId: string;
+};
+
 export const VirtualKeyboardButtons = NamedFC<VirtualKeyboardButtonsProps>(
     'VirtualKeyboardButtons',
     props => {
-        const getArrowButton = (text: string, className?: string) => {
+        const getArrowButton = (text: string, onClick: () => void, className?: string) => {
             return (
                 <Button className={styles.button} primary>
```

Comment:
You need to wire in the onClick handler...
```suggestion
                <Button onClick={onClick} className={styles.button} primary>
```